### PR TITLE
LPS-98417 removing form field tests from ProjectTemplatesTest and creating them inside of the form field template project

### PR DESCRIPTION
**Pull request by gabrielibson**

### CODE REVIEW

**Comment by gabrielibson** [comment]
Related issue: https://issues.liferay.com/browse/LPS-98417

**Comment by gabrielibson** [comment]
ci:test:sf

**Comment by gabrielibson** [comment]
ci:test:relevant

**Comment by gabrielibson** [comment]
ci:test:relevant

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/sdk/project-templates/project-templates-form-field/src/main/resources/archetype-resources/src/main/resources/META-INF/resources/__artifactId__.es.js [hunk 1]
```diff
@@ -1,6 +1,14 @@
+#parse ("definitions.vm")
```
Comment (natocesarrego) [inline modules/sdk/project-templates/project-templates-form-field/src/main/resources/archetype-resources/src/main/resources/META-INF/resources/__artifactId__.es.js]: Hey @gabrielibson, you just need to add this line if you'll use some variable of this file, like `formFieldTypeName`, but apparently this is not the case, right? If so, please remove this line.

File: modules/sdk/project-templates/project-templates-form-field/src/main/resources/archetype-resources/src/main/resources/META-INF/resources/__artifactId__Register.soy [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#parse ("definitions.vm")
+{namespace ${className}Register}
+
+{deltemplate PageRenderer.RegisterFieldType variant="'${formFieldTypeName}'"}
+    {call ${className}.render data="all" /}
+{/deltemplate}
```
Comment (natocesarrego) [inline modules/sdk/project-templates/project-templates-form-field/src/main/resources/archetype-resources/src/main/resources/META-INF/resources/__artifactId__Register.soy]: @gabrielibson, on your commit message, you say that you're updating the `build.gradle`, but where is this update on this commit?

File: modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java [hunk 3]
```diff
@@ -846,6 +843,91 @@ public void testBuildTemplateFormField71WithHyphen() throws Exception {
...
+			temporaryFolder, "form-field", "foo-bar", "com.test", mavenExecutor,
+			"-DclassName=FooBar", "-Dpackage=foo-bar",
+			"-DliferayVersion=7.2.0");
+
+		testContains(
+			mavenProjectDir, "bnd.bnd", "-contract: JavaPortlet,JavaServlet");
+
+		_buildProjects(gradleProjectDir, mavenProjectDir);
+	}
+
```
Comment (natocesarrego) [inline modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java]: @gabrielibson, ideally, you should make these tests check only the changes introduced by the updates demanded to make the form-field template support 7.2. Can you please revisit these tests with this in mind?

**Comment by natocesarrego** [comment]
Pull request submitted to https://github.com/gabrielibson/liferay-portal/pull/7. [See changes here.](https://github.com/natocesarrego/liferay-portal/compare/0aa12bb1c803~1...0aa12bb1c803) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**